meson_options.txt

```diff
@@ -7,6 +7,7 @@ option('introspection', type : 'boolean', value : true, description : 'generate
 option('lvfs', type : 'boolean', value : true, description : 'enable LVFS remotes')
 option('man', type : 'boolean', value : true, description : 'enable man pages')
 option('libarchive', type : 'boolean', value : true, description : 'enable libarchive support')
+option('libflashrom', type : 'boolean', value : true, description : 'enable libflashrom support')
```

Comment:
I find it hard to make the argument of default of this to be `true` when `plugin_flashrom` is default to `false`.

Furthermore, if this PR ends up directionally being correct, I think that `plugin_flashrom` should just become `flashrom`.  In switching to `flashrom` it would be "support for flashrom based devices".  Similar to how we have support for gusb/guedev/bluez but don't prefix them with `lib`